libsolidity/experimental/codegen/IRGeneratorForStatements.cpp

```diff
@@ -47,9 +48,70 @@ std::string IRGeneratorForStatements::generate(ASTNode const& _node)
 }
 
 
-namespace
+namespace solidity::frontend::experimental
 {
```

Comment:
If you only need this locally here in this cpp file, using the anonymous namespace ``namespace { ... }`` is usually what you do - but I actually didn't mean for you to move ``stackSize`` out of ``IRGeneratorForStatements`` at all - it's perfectly fine as a static function in there - my point in the other comment was just that within ``IRGeneratorForStatements``, you can still call it as ``stackSize(...)`` and don't need to use ``IRGeneratorForStatements::stackSize(...)``, nothing more :-).

Reply:
Oh right. I thought I should move it based on this comment here: https://github.com/ethereum/solidity/pull/14620#discussion_r1472810562. Fixed ;)

Reply:
I just meant moving the implementation to the cpp file instead of keeping it inlined in the header :-).